server/java/src/main/java/com/godaddy/asherah/grpc/DynamoDbConfig.java

```diff
@@ -0,0 +1,42 @@
+package com.godaddy.asherah.grpc;
+
+public class DynamoDbConfig {
```

Comment:
the members of this class shouldn't be static. instead make them final and initialize via the constructor.

All the other getter will not need to be static then.